WindowManager/app/src/main/java/com/example/windowmanagersample/DisplayFeaturesActivity.kt

```diff
@@ -84,7 +85,6 @@ class DisplayFeaturesActivity : BaseSampleActivity() {
         layoutInfo?.let { windowLayoutInfo ->
             stateStringBuilder.append(getString(R.string.window_layout))
                 .append(": ")
-                .append(windowLayoutInfo)
 
             // Add views that represent display features
             for (displayFeature in windowLayoutInfo.displayFeatures) {
```

Comment:
does getLayoutParamsForFeatureInFrameLayout return the right values with the new API changes?

Reply:
I've not seen anything that requires an update in `getLayoutParamsForFeatureInFrameLayout` or `getFeaturePositionInViewRect` with the newer WindowManager versions.